apps/backoffice-v2/.env.example

```
@@ -3,3 +3,4 @@ VITE_API_KEY=demo
 VITE_AUTH_ENABLED=true
 VITE_MOCK_SERVER=false
 VITE_POLLING_INTERVAL=10
+VITE_ASSIGNMENT_POLLING_INTERVAL=5
```

Please mention the unit you use, in this case, seconds.
https://ruudvanasseldonk.com/2022/03/20/please-put-units-in-names
```suggestion
VITE_ASSIGNMENT_POLLING_INTERVAL_IN_SECONDS=5
```